server/CMakeLists.txt

```diff
@@ -1,22 +1,31 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 2.6)
 project(ins_server)
 
+set(CMAKE_C)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread")
+
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 
+find_package (LibXml2)
+
 set(SOURCE_FILES
     src/main.cpp
     src/ins_service.cpp
     src/data_store.cpp
     src/localization.cpp
     src/lib_wrapper.cpp
+    src/WifiNode.c
+    src/WifiAccessPointLocalConfig.c
  )
 
 include_directories(
     ${CMAKE_FIND_ROOT_PATH}/include
     ${CMAKE_CURRENT_SOURCE_DIR}/include
-    ${CMAKE_CURRENT_SOURCE_DIR}/include/external )
+    ${CMAKE_CURRENT_SOURCE_DIR}/include/external)
+
+link_directories(${CMAKE_CURRENT_SOURCE_DIR}/libxml2)
 
-link_directories(${CMAKE_FIND_ROOT_PATH}/lib)
```

Comment:
What was `lib` doing before and now it's not necessary?

Reply:
It was not necessary before. And will actually not be necessary now. That directory did not exist.